src/js/analyses/components/Analyses.tsx

```diff
@@ -1,34 +1,15 @@
 import React, { useEffect } from "react";
 import { connect } from "react-redux";
-import { Route, Switch, useRouteMatch } from "react-router-dom";
-import { findHmms } from "../../hmm/actions";
-import { listReadyIndexes } from "../../indexes/actions";
-
+import { Route, Switch } from "react-router-dom";
 import { LoadingPlaceholder } from "../../base";
-import { findAnalyses } from "../actions";
+import { listReadyIndexes } from "../../indexes/actions";
+import AnalysesList from "./AnalysisList";
 import AnalysisDetail from "./Detail";
-import AnalysesList from "./List";
-
-interface AnalysesProps {
-    loading: boolean;
-    onFindAnalyses: (sampleId: string) => void;
-    onFindHmms: () => void;
-    onListReadyIndexes: () => void;
-}
-
-interface MatchTypes {
-    /** The sample id */
-    sampleId: string;
-}
-
-function Analyses({ loading, onFindAnalyses, onFindHmms, onListReadyIndexes }: AnalysesProps) {
-    const sampleId = useRouteMatch<MatchTypes>().params.sampleId;
 
+function Analyses({ loading, onListReadyIndexes }) {
     useEffect(() => {
-        onFindAnalyses(sampleId);
-        onFindHmms();
         onListReadyIndexes();
```

Comment:
b/c of createAnalysis?

Reply:
Yes, since we wanted to leave it alone I figured we can do it once the conflicts merge.